### PR TITLE
Apply the new design for organization view

### PR DESCRIPTION
**Pull request by bouguila**
changed design of single organization view to meet Starcounter/RebelsLounge#121

### CODE REVIEW

File: src/People/wwwroot/People/css/people.css [hunk 1]
```diff
@@ -252,4 +252,33 @@
...
+}
+.people-anchor-no-underline > div.people-person-tag {
+    -moz-transition: all .2s ease-in;
+    -o-transition: all .2s ease-in;
+    -webkit-transition: all .2s ease-in;
+    transition: all .2s ease-in;
+    color:#111111;   
+}
+
+    .people-anchor-no-underline > div.people-person-tag:hover {
```
Comment (Mackiovello) [inline src/People/wwwroot/People/css/people.css]: Is this kind of indentation something we encourage? [CSS beautifier](http://www.cleancss.com/css-beautify/) doesn't do it this way.

File: src/People/wwwroot/People/css/people.css [hunk 2]
```diff
@@ -252,4 +252,33 @@
...
+        color: #FFFFFF;
+        cursor: pointer;
+        background-color: #C0C0C0;
+    }
+.people-organization-parent-control {
+    display:flex;
+    flex-direction:row;
```
Comment (Mackiovello) [inline src/People/wwwroot/People/css/people.css]: Please be consisted with spacing between after the colon

File: src/People/wwwroot/People/viewmodels/OrganizationPage.html [hunk 3]
```diff
@@ -94,36 +92,31 @@ <h2 slot="people/organization-heading">Organization details</h2>
...
             }
+                .people-organization-main-infos > div {
+                    margin-bottom: 20px;
+                }
 
-            .people-infogroup__label {
-                margin-bottom: 20px;
-                width: 100%;
+            .people-organization-footer{
+                grid-area:footer;
```
Comment (Mackiovello) [inline src/People/wwwroot/People/viewmodels/OrganizationPage.html]: Spacing, as mentioned above

File: src/People/wwwroot/People/css/people.css [hunk 4]
```diff
@@ -252,4 +252,33 @@
 }
 span.people-icon-color {
     color: #2196f3;
-}
\ No newline at end of file
+}
+a.people-anchor-no-underline {
+    text-decoration: none;
+}
+.people-anchor-no-underline > div.people-person-tag {
```
Comment (Mackiovello) [inline src/People/wwwroot/People/css/people.css]: I would not recommend this kind of selector, see http://getbem.com/faq/#encapsulating-tag-selector

File: src/People/wwwroot/People/viewmodels/OrganizationPage.html [hunk 5]
```diff
@@ -137,92 +130,99 @@ <h2 slot="people/organization-heading">Organization details</h2>
             }
 
             .people-heading {
+                grid-area:header;
```
Comment (Mackiovello) [inline src/People/wwwroot/People/viewmodels/OrganizationPage.html]: Spacing

**Comment by bouguila** [comment]
ping @Mackiovello

**Comment by Mackiovello** [comment]
I don't know how it's supposed to look, but the code looks fine. Do you have a mockup that I can compare to?

**Comment by Mackiovello** [comment]
This is not directly relevant to this PR, but I would love to see a more compact view, like this:
capture3

**Comment by bouguila** [comment]
@Mackiovello we don't have a mockup for the organization's view but here is the general mockup https://github.com/Starcounter/RebelsLounge/issues/121, that this view should be the same "look & feel"

**Comment by Mackiovello** [comment]
Got it! I think it looks fine

**Comment by bouguila** [comment]
thanks @Mackiovello

**Pull request merged**